test/IRGen/assoctypepath.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %target-swift-frontend -primary-file %s -emit-ir | %FileCheck %s
```

Comment:
This will fail because there are no CHECK: directives -- can you add some for the mangled symbols?

Reply:
oh, thanks! Yeah, that's what I wanted but apparently not did